packages/nimble-components/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "19.0.1",
   "description": "Styled web components for the NI Nimble Design System",
   "scripts": {
-    "build": "npm run generate-icons && npm run build-components && npm run bundle-components && npm run generate-scss && npm run build-storybook",
+    "build": "wireit",
```

Comment:
When I run local builds both with a wireit cache and without wireit at all, I'm seeing the one with wireit be a bit faster (maybe 10s out of about a minute?) 👍

But the wireit build is a lot noisier in the console, with tons of messages like this which are printed during the "dist/esm/all-components.js → dist/all-components-bundle.min.js..." step:
```
[webpack.Progress] 92% sealing asset processing SourceMapDevToolPlugin 9043.748376a8.iframe.bundle.js generate SourceMap
```

Any idea why they started happening? Can they be quieted?

Reply:
That actually comes from the storybook build. It used to update the progress line in-place, so there wasn't as much noise. We can get rid of it by passing a `--quiet` flag.

Reply:
That change successfully quieted that build output and we're back to our previous level. 🎉

I did some more testing because I was surprised that we weren't saving more time. Here's what I found when running commands from the root:
`npm run build -w @ni/nimble-components` is now super fast
```
> @ni/nimble-components@19.0.1 build
> wireit

✅ [generate-icons] Already fresh
✅ [build-components] Already fresh
✅ [bundle-components] Already fresh
✅ [generate-scss] Already fresh
✅ [build-storybook] Already fresh
✅ [build] No command to execute
```

But `npm run build` isn't detecting that those commands are "fresh" and it's actually re-running most of them:
```
> @ni/nimble-components@19.0.1 build
> wireit

🏃 [generate-icons] Running command "npm run generate-icons:bundle && npm run generate-icons:run"

> @ni/nimble-components@19.0.1 generate-icons:bundle
> rollup --bundleConfigAsCjs --config build/generate-icons/rollup.config.js


/Users/jesse/Documents/GitHub/nimble2/packages/nimble-components/build/generate-icons/source/index.js → build/generate-icons/dist/index.js...
created build/generate-icons/dist/index.js in 72ms

> @ni/nimble-components@19.0.1 generate-icons:run
> node build/generate-icons/dist/index.js

........

Finished writing all-icons file
✅ [generate-icons] Executed successfully
🏃 [build-components] Running command "tsc --build --pretty ./tsconfig.json"
✅ [build-components] Executed successfully
🏃 [bundle-components] Running command "rollup --bundleConfigAsCjs --config"
🏃 [generate-scss] Running command "npm run generate-scss:bundle && npm run generate-scss:run"
🏃 [build-storybook] Running command "storybook build -o dist/storybook --webpack-stats-json --quiet"

.......

✅ [bundle-components] Executed successfully
info => Copying static files: /Users/jesse/Documents/GitHub/nimble2/packages/nimble-components/.storybook/public at /Users/jesse/Documents/GitHub/nimble2/packages/nimble-components/dist/storybook/
info => Preview built (19 s)
WARN Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
WARN Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
info => preview stats written to /Users/jesse/Documents/GitHub/nimble2/packages/nimble-components/dist/storybook/preview-stats.json
info => Output directory: /Users/jesse/Documents/GitHub/nimble2/packages/nimble-components/dist/storybook
✅ [build-storybook] Executed successfully
✅ [build] No command to execute
```

Does that match what you're seeing? I guess it makes sense: if we're not using wireit for `nimble-tokens` yet then the inputs to the `nimble-components` build will always be dirty. Maybe we should take a less piecemeal approach and just use wireit everywhere? I was worried about reliability when I suggested we be more careful, but perhaps we could mitigate that with local dev testing.

Reply:
I wonder if it's related to this comment: https://github.com/ni/nimble/pull/1267#discussion_r1202982774

Where the two commands ``npm run generate-scss:bundle` and  `npm run generate-scss:run` have files and outputs that are coupled together. Splitting them up may help, but should be done either way to make understanding the inputs and outputs of those separate scripts clearer

Reply:
I've added Wireit for the Angular, Blazor, and Tokens workspaces. There are still a couple others small ones I didn't bother with.

Reply:
Cached builds are super fast now for both the mono build (5 seconds!) and the individual workspace builds. Awesome dev experience! And I verified that changing a couple different files resulted in the right set of things rebuilding (other than some cases that I left comments about).